docs/_future/getting_started_future.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Getting started
+order: 1
+slug: getting-started-future
+lang: en
+layout: tutorial
+permalink: /tutorial/getting-started-future
+---
+
+## Getting started <span class="label-beta">BETA</span>
+
+<div class="section-content">
+This guide will cover how to get started with your next-gen platform using Bolt for Python by setting up the Slack CLI and also installing the required dependencies.
+</div>
+
+---
+
+### What is the next-generation platform? {#next-gen}
+
+You can find all the answers to this question on Slack's <a href="https://api.slack.com/future/intro" target="_blank">official introduction page.</a>
```

Comment:
Thoughts on removing this "What is the next-generation platform" and just including another sentence under  the main "Getting Started" header? 

Bolt-js includes a Limitations subheader which notes that we don't support Datastores or deploying to Slack infrastructure. We should have something like this, but I think we can actually move the content somewhere else, like it's own header.

https://slack.dev/bolt-js/future/getting-started

Reply:
Yeah this sounds good 💯  Ill include the limitation header and talk with @hello-ashleyintech about potentially centralizing this info somewhere